nle/scripts/ttyplay.py

```diff
@@ -10,6 +10,7 @@
 import struct
 import termios
 import time
+from nle.nethack.actions import _ACTIONS_DICT
```

Comment:
I don't like this as it makes ttyplay.py nethack-only all of a sudden. Better to add another script and import the relevant functions.

Reply:
do you mean `nle` only?

Reply:
do you mean now you cant just take the script and play any old ttyrec, away from nle?